dali/operators/reader/loader/webdataset/tar_utils.cc

```diff
@@ -165,6 +165,10 @@ int64_t TarArchive::TellArchive() const {
   return current_header_;
 }
 
+int64_t TarArchive::HeaderSize() const {
+  return stream_->Tell() - readoffset_ - current_header_;
```

Comment:
Does it assume that we are at the file boundary? Can call to `TarArchive::Read` affect that assumption?

Reply:
no, it doesn't